ds3-autogen-c/src/main/resources/templates/request-templates/AmazonS3InitRequest.ftl

```diff
@@ -3,7 +3,7 @@
 <#--   Input: Source object                    -->
 <#-- ***************************************** -->
 <#list getRequests() as requestEntry>
-ds3_request* s3_init_${requestEntry.getRequestHelper().getNameRootUnderscores(requestEntry.getName())}(const char* bucket_name) {
+ds3_request* s3_init_${requestEntry.getRequestHelper().getNameRootUnderscores(requestEntry.getName())}(${requestEntry.getRequestHelper().getAmazonS3InitParams(requestEntry.getRequiredArguments())}) {
```

Comment:
This is not the convention that we've been using in .net or in java.  We should match the conventions as much as possible.  The .net and java code add spectras3 at the end of the command names.